src/graph/graphcontext.h

```diff
@@ -54,6 +47,15 @@ int GraphContext_AddIndex(GraphContext *gc, const char *label, const char *prope
 // Remove and free an index
 int GraphContext_DeleteIndex(GraphContext *gc, const char *label, const char *property);
 
+// Add a single node to all indices its properties match
+void GraphContext_AddNodeToIndices(GraphContext *gc, LabelStore *store, Node *n);
```

Comment:
Either have all Indices functions here get an optional store pointer, or remove store from the signature and find store depending on rather or not node has a label,
I think I prefer former approach as it gives just a little bit more room for optimisation
What do you think?